tests/testthat/test-translate-sql.R

```diff
@@ -26,11 +26,11 @@ test_that("between translated to special form (#503)", {
 
 test_that("is.na and is.null are equivalent", {
   # Needs to be wrapped in parens to ensure correct precedence
-  expect_equal(translate_sql(is.na(x)), sql("((`x`) IS NULL)"))
-  expect_equal(translate_sql(is.null(x)), sql("((`x`) IS NULL)"))
+  expect_equal(translate_sql(is.na(x)), sql("(`x` IS NULL)"))
```

Comment:
I think this probably needs a new test for `translate_sql(is.na(x + y))` to capture the spirit of the original test.